universum_log_collapser/e2e/universum_live_log_to_html.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+import os
+import subprocess
+import re
+
+
+def run_universum():
+    os.environ["PYTHONUNBUFFERED"] = "1"
+    cmd = """universum --clean-build --vcs-type=none --server-type=local --out-type=term 
+        --launcher-output=console --file-source-dir=. --launcher-config-path=./universum_config/configs.py"""
+    popen = subprocess.Popen(cmd.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    assert popen.wait() == 0, popen.stderr.read().decode()
```

Comment:
Please double check that exit code is really returned.
If steps in the config fail, universum has to return error.

Reply:
`universum ${valid_params}` returns 0
`python -m universum ${valid_params}` returns 0
`universum ${invalid_params}` returns 2